config/configtls/go.mod

```diff
@@ -12,11 +12,11 @@ require (
 
 require (
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/google/go-tpm-tools v0.4.4 // indirect
+	github.com/google/go-tpm-tools v0.4.5 // indirect
 	github.com/kr/text v0.2.0 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	golang.org/x/crypto v0.19.0 // indirect
```

Comment:
I wasn't able to pass `make golint` locally without this change, but if https://github.com/open-telemetry/opentelemetry-collector/pull/12946 should also resolve the issue. I will rebase once that's merged.